src/common.cc

```diff
@@ -389,8 +389,11 @@ double cpu_barval(struct text_object *obj) {
 
 #define PRINT_HR_GENERATOR(name)                                        \
   void print_##name(struct text_object *obj, char *p, int p_max_size) { \
-    (void)obj;                                                          \
-    human_readable(info.name * 1024, p, p_max_size);                    \
+    int base = 1024; \
```

Comment:
Can you move this logic into `human_readable` instead of doing this? It fails the DRY test.

Reply:
@brndnmtthws Not all `human_readable()` calls multiply their variables, so how do we know when to use the `base` system ?

Reply:
passing int of 0 and 1 as formal parameter to tell when to use perharps ?